bundles/af-core/src/test/resources/form/formcontainer/exporter-formcontainerv2.json

```diff
@@ -10,6 +10,7 @@
     },
     "fd:path": "/content/forms/af/demo/jcr:content/formcontainerv2",
     "fd:schemaType": "BASIC",
+    "fd:roleAttribute":null,
```

Comment:
Please add test case where this property in stored in jcr as fd:roleAttribute and the same is present in the JSON

Reply:
Added the test case for the JCR node having property  fd:roleAttribute.

Reply:
I don't see the test case yet

Reply:
I have added the test cases for checking the attribute in the json and also checked the jcr node value of that attribute.